addons/stock_picking_batch/views/stock_picking_views.xml

```diff
@@ -21,7 +21,7 @@
                             ('picking_type_id', '=', picking_type_id),
                             ('picking_type_id', '=', False),
                     ]"
-                    readonly="state in ['cancel', 'done']"
+                    readonly="1"
```

Comment:
don't forget to remove the domain in that case

Reply:
Domain removed, thanks!